src/error.rs

```diff
@@ -18,22 +18,20 @@ pub type EvalResult<T> = Result<T, EvalError>;
 impl Error for EvalError {
     fn description(&self) -> &str {
         match *self {
-            EvalError::DanglingPointerDeref =>
-                "dangling pointer was dereferenced",
-            EvalError::InvalidBool =>
-                "invalid boolean value read",
-            EvalError::InvalidDiscriminant =>
-                "invalid enum discriminant value read",
-            EvalError::PointerOutOfBounds =>
-                "pointer offset outside bounds of allocation",
-            EvalError::ReadPointerAsBytes =>
-                "a raw memory access tried to access part of a pointer value as raw bytes",
-            EvalError::ReadBytesAsPointer =>
-                "attempted to interpret some raw bytes as a pointer address",
-            EvalError::InvalidPointerMath =>
-                "attempted to do math or a comparison on pointers into different allocations",
-            EvalError::ReadUndefBytes =>
-                "attempted to read undefined bytes",
+            EvalError::DanglingPointerDeref => "dangling pointer was dereferenced",
+            EvalError::InvalidBool => "invalid boolean value read",
+            EvalError::InvalidDiscriminant => "invalid enum discriminant value read",
+            EvalError::PointerOutOfBounds => "pointer offset outside bounds of allocation",
+            EvalError::ReadPointerAsBytes => {
+                "a raw memory access tried to access part of a pointer value as raw bytes"
+            }
+            EvalError::ReadBytesAsPointer => {
+                "attempted to interpret some raw bytes as a pointer address"
+            }
+            EvalError::InvalidPointerMath => {
+                "attempted to do math or a comparison on pointers into different allocations"
+            }
+            EvalError::ReadUndefBytes => "attempted to read undefined bytes",
```

Comment:
Yep, I don't like it either because it messes with consistency.

(Setting `format_strings = false` seems to solve this, by the way, but I didn't want to add a `rustfmt.toml` in this PR.)